build-all-branches.bash

```diff
@@ -1,9 +1,28 @@
 #!/usr/bin/env bash
 test -f "$HOME/.jdk8" && . $HOME/.jdk8
 
+function ideaBranch {
```

Comment:
multiline fixme highlighting was added in 183.x. BashSupport is still supporting 145.x and later.
I wanted to have a test case. This test is in a separate source root `test-183` (instead of `test`). The gradle build adds this root if running with 183.x. 
This script compiles and runs tests with all supported versions and passes the current branch to gradle which needs this to activate the source root. (see the change to `build.gradle`)